src/redux/reducers/index.js

```diff
@@ -0,0 +1,10 @@
+/*import { combineReducers } from "redux";
```

Comment:
I forgot about the combineReducers import, that will keep things organized, nice addition.